### PR TITLE
Add passwords feature

### PR DESCRIPTION
**Pull request by sachinwalunjakar**
- check security
- please review changes in file1

### CODE REVIEW

File: file1.txt [hunk 1]
```diff
@@ -1 +1 @@
-full feature implemented
```
Comment (walunjakarsachinedu) [inline file1.txt]: For adding password feature, you have remove previously added feature.
So, please re-add that feature.

**Review by walunjakarsachinedu — CHANGES_REQUESTED** [review]

File: file1.txt (hunk 1 above)
Comment (sachinwalunjakar) [reply file1.txt]: Yes, I will re-add that feature. That for reviewing

**Review by walunjakarsachinedu — CHANGES_REQUESTED** [review]
Everything is good, now you merge changes.

**Review by walunjakarsachinedu — APPROVED** [review]
Everything is good, now you can merge the changes.

**Pull request merged**